.devcontainer/Dockerfile

```diff
@@ -1,32 +1,3 @@
-FROM mcr.microsoft.com/devcontainers/base:ubuntu-20.04
-
-RUN apt update \
-    && DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends \
-        clang-format \
-        clang-tidy \
-        cmake \
-        g++ \
-        gcc \
-        gdb \
-        libboost-container-dev \
-        libgtest-dev \
-        make \
-    && rm -rf /var/lib/apt/lists/*
-
-# Install units library
-RUN git clone --depth 1 --branch v2.3.3 https://github.com/nholthaus/units.git /tmp/units \
-    && cd /tmp/units \
-    && cmake -B build -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTS=OFF -DBUILD_DOCS=OFF \
-    && cmake --build build \
-    && cmake --install build \
-    && cd / \
-    && rm -rf /tmp/units
-
-# Install Eigen3
-RUN git clone --depth 1 --branch 3.4.0 https://gitlab.com/libeigen/eigen.git /tmp/eigen \
-    && cd /tmp/eigen \
-    && cmake -B build -DCMAKE_BUILD_TYPE=Release -DEIGEN_BUILD_TESTING=OFF -DEIGEN_BUILD_DOC=OFF \
-    && cmake --build build \
-    && cmake --install build \
-    && cd / \
-    && rm -rf /tmp/eigen
+FROM ghcr.io/usdot-fhwa-stol/carma-builds-x64:focal
```

Comment:
is this going to work still? I think now we need carma-system-4.5.0 right?

Reply:
I guess this one is .devcontainer so it doesnt matter prolly

Reply:
from conversation on https://github.com/usdot-fhwa-stol/carma-streets/pull/411
I am still not sure if stays the same or needs to have carma-system tag included

Reply:
It is devcontain so it does not matter